lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -198,6 +198,31 @@ def get_release_url(repository:, tag_name:)
         nil
       end
 
+      # Publishes an existing GitHub Release still in draft mode. Note that this will search only within the first page of releases, as returned by the GitHub API.
+      #
+      # @param [String] repository The repository name, including the organization (e.g. `wordpress-mobile/wordpress-ios`)
+      # @param [String] name The name of the release to publish.
+      # @param [Boolean] prerelease Indicates if this should be created as a pre-release (i.e. for alpha/beta)
+      #
+      # @return [String] URL of the corresponding GitHub Release
+      #
+      def publish_release(repository:, name:, prerelease: false)
+        releases = client.releases(repository)
+        release = releases.find { |r| r.name == name }
+
+        UI.user_error!("No release found with name #{name}") unless release
+
+        client.update_release(
+          release.url,
+          {
+            draft: false,
+            prerelease: prerelease
+          }
```

Comment:
If you end up going with my suggestion of supporting `:unchanged` in the action's parameter, and thus `nil` for the `prerelease` parameter of this helper, you might need to call `.compact` on this hash to remove the potential `prerelease: nil` value from it when calling the API

Reply:
👍 
Updated on f9fa87c2e3ba199b560ee25ba153a90cfe2e9013